user_templates/printer.cfg

```diff
@@ -166,6 +166,12 @@
 
 # ---------------------------------------------------------- ERCF ----
 ### ------------------------------------------------------------------
+### Klippain is optimized to be used with ERCF Happy Hare https://github.com/moggieuk/ERCF-Software-V3
+### All this files created by install script of Happy Hare must be include
```

Comment:
"All these files are created by the Happy Hare install script and must be included"

Reply:
done